modules/cudf/src/series.ts

```diff
@@ -194,6 +195,52 @@ export class AbstractSeries<T extends DataType = any> {
     return this.__construct(this._col.gather(selection._col));
   }
 
+  /**
+   * Scatters single value into this Series according to provided indices.
+   *
+   * @param value A column of values to be scattered in to this Series
+   * @param indices A column of integral indices that indicate the rows in the this Series to be
+   *   replaced by `value`.
+   * @param check_bounds Optionally perform bounds checking on the indices and throw an error if any
+   *   of its values are out of bounds (default: false).
+   * @param memoryResource An optional MemoryResource used to allocate the result's device memory.
+   */
+  scatter(value: T['scalarType'],
+          indices: Series<Int32>|number[],
+          check_bounds?: boolean,
+          memoryResource?: MemoryResource): void;
+  /**
+   * Scatters a column of values into this Series according to provided indices.
+   *
+   * @param value A value to be scattered in to this Series
+   * @param indices A column of integral indices that indicate the rows in the this Series to be
+   *   replaced by `value`.
+   * @param check_bounds Optionally perform bounds checking on the indices and throw an error if any
+   *   of its values are out of bounds (default: false).
+   * @param memoryResource An optional MemoryResource used to allocate the result's device memory.
+   */
+  scatter(values: Series<T>,
+          indices: Series<Int32>|number[],
+          check_bounds?: boolean,
+          memoryResource?: MemoryResource): void;
+
+  scatter(source: Series<T>|T['scalarType'],
+          indices: Series<Int32>|number[],
+          check_bounds?: boolean,
```

Comment:
Should this default to libcudf's default?
```suggestion
          check_bounds = false,
```